torontoadvotech/client
# Ws 15

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Added new events page, with additional events success modal for form submits, and single event page for more information about particular event. All event copy and images related to events is in separate events.ts file.

## Related Issue



## Acceptance Criteria

[WS-15](https://torontoadvotech.atlassian.net/browse/WS-15)

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Testing Steps / QA Criteria

Navigate to /events page and ensure that page and features match those in mockup.
